builder/hcloud/config.go

```diff
@@ -47,6 +47,7 @@ type Config struct {
 	SSHKeys        []string          `mapstructure:"ssh_keys"`
 	SSHKeysLabels  map[string]string `mapstructure:"ssh_keys_labels"`
 	Networks       []int64           `mapstructure:"networks"`
+	PublicIPv4     string            `mapstructure:"public_ipv4"`
```

Comment:
What do you think about using `public_ipv4_address` instead? Most if not all of our options takes an ID or a name as parameter. Using the IP address here, without using the `_address` suffix, would diverge from this convention that the users would expect.